doc/pylint.rst

```diff
@@ -1,36 +1,97 @@
-####################
-Static Code Analysis
-####################
+#############
+Code Analysis
+#############
 
-The **Static Code Analysis** pane detects style issues, bad practices, potential bugs, and other quality problems in your code, all without having to actually execute it.
-Spyder's static analyzer is powered by the best in class `Pylint`_ back-end, which can intelligently detect an enormous and customizable range of problem signatures.
+The **Code Analysis** pane detects style issues, bad practices, potential bugs, and other quality problems in your code, all without having to actually execute it.
+Based on this, it gives you a global evaluation of your code reflected on a score from 0 to 10.
```

Comment:
I still thinks it would be necessary to specify the range of the score, otherwise it's hard to understand. Is it from -10 to 10? Or how does it work?

Reply:
@ccordoba12 thoughts?

Reply:
Yeah, I wanted to include it which is why I did some testing to find out, but

> There isn't actually a clear lower bound, scores can go negative if you have enough warnings and errors (I've seen it in the wild a number of times before, and I was able to get >-50 in some quick testing.

So while there's an upper bound (which is already explicitly shown in the UI), there's no lower bound so I'm not sure how to explain it clearly and concisely especially given the context of a high-level summary of the whole pane, so I feel the only thing we can do is cut it.

Reply:
I played around with different ways of putting it for while but just couldn't find a clear and concise way of putting it. If one of you can, I guess I'm fine with it but I'm really not sure how since its not just a neat defined range.